content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -275,8 +275,35 @@ will choose to evict pods of lowest Priority first.
 
 If you are running a [static pod](/docs/concepts/workloads/pods/#static-pods)
 and want to avoid having it evicted under resource pressure, set the
-`priority` field for that Pod directly. Static pods do not support the
-`priorityClassName` field.
+`priorityClassName` field for that Pod directly. 
+
+For example, the following configuration sets the priorityClassName for a static pod as `xyz-priority`. 
+
+```yaml
+kubectl get priorityclass
+NAME                      VALUE        GLOBAL-DEFAULT   AGE
+system-cluster-critical   2000000000   false            2m2s
+system-node-critical      2000001000   false            2m2s
+xyz-priority              1000000      false            47s
```

Comment:
(nit) We prefer to [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)

Reply:
Fixed nit